src/components/Cart/__tests__/__snapshots__/Cart.test.js.snap

```diff
@@ -2,19 +2,39 @@
 
 exports[`COMPONENT - Cart renders Cart component 1`] = `
 .c0 {
-  padding-top: 3rem;
   padding-bottom: 12rem;
 }
 
+.c1 {
```

Comment:
To reduce noise in snapshots be free to mock this component in test.
So in _Cart.test.js_ add `jest.mock('../../components/InfoBanner', () => 'InfoBanner')`

Reply:
I didn't know about that, good thing to learn! Added, will be part of updated PR

Reply:
👍